src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+---
+layout: post
+title: Registering a web-based payment app
+subhead: |
+  Learn how to configure a web-based payment app on registration.
```

Comment:
```suggestion
  Learn how to configure a web-based payment app during service worker registration.
```

Reply:
This "registration" is for a web-based payment app. Not necessarily about a service worker.